examples/03-Maxwell/Maxwell3D_Team3_bath_plate.py

```diff
@@ -29,19 +38,18 @@
 
 project_name = "COMPUMAG"
 design_name = "TEAM 3 Bath Plate"
-Solver = "EddyCurrent"
-desktop_version = "2023.2"
+solver = "EddyCurrent"
+desktop_version = "2024.1"
```

Comment:
@gmalinve as far as I know CICD still host only 2023.2. @Samuelopez-ansys  is it correct?

Reply:
Correct!